referrer-policy/no-referrer-policy/no-referrer-policy.subresource.py

```diff
@@ -0,0 +1,16 @@
+import os, json
+
+def main(request, response):
+    script_directory = os.path.dirname(os.path.abspath(__file__))
+    template_basename = "no-referrer-policy.subresource.template.html"
+    template_filename = os.path.join(script_directory, template_basename);
+
+    with open(template_filename) as f:
+        template = f.read()
+
+    headers_as_json = json.dumps(request.headers)
+    replacement_tag = "/* <SERVER_REQUEST_HEADERS> */"
+    exported_headers = "var SERVER_REQUEST_HEADERS = " + headers_as_json + ";"
+    rendered_html = template.replace(replacement_tag, exported_headers)
```

Comment:
Hrm. This is a bit ad hoc. It's fine for right now, but I think it would be better to use a real templating library, rather than poking at the file with a replacement method.